pom.xml

```diff
@@ -53,7 +53,7 @@
         <dep.tempto.version>175</dep.tempto.version>
         <dep.gcs.version>2.0.0</dep.gcs.version>
         <dep.errorprone.version>2.3.4</dep.errorprone.version>
-        <rubix.version>0.3.9</rubix.version>
+        <rubix.version>0.3.10-SNAPSHOT</rubix.version>
```

Comment:
undo. We don't want to depend on snapshot versions

Reply:
I'll update to `0.3.10` on merge and release of https://github.com/qubole/rubix/pull/389

Reply:
Updated to `0.3.10`